packages/vue/__tests__/createVuexPlugin.js

```diff
@@ -0,0 +1,22 @@
+import createVuexPlugin from '../src/createVuexPlugin';
```

Comment:
Filename misses test. Should be createVuexPlugin.test.js.

Reply:
fixed